### PR TITLE
feat: disable scrubbing for known AWS services

### PR DESCRIPTION
**Pull request by nirLumigo**
Before the feature:
image
image

After:
image
image

### CODE REVIEW

File: src/utils/payloadStringify.test.js [hunk 1]
```diff
@@ -182,4 +182,45 @@ describe('payloadStringify', () => {
...
+
+  test('payloadStringify -> skipScrubPath -> Nested with array', () => {
+    const payload = { Records: [{ object: { key: 'value' } }, { object: { key: 'value' } }] };
+    const result = payloadStringify(payload, 1024, ['Records', [], 'object', 'key']);
+    expect(result).toEqual(JSON.stringify(payload));
+  });
+
+  test('payloadStringify -> skipScrubPath -> Doesnt affect other paths', () => {
+    const result = payloadStringify({ o: { key: 'value', password: 'value' } }, 1024, ['o', 'key']);
+    expect(result).toEqual(JSON.stringify({ o: { key: 'value', password: '****' } }));
```
Comment (doriaviram) [inline src/utils/payloadStringify.test.js:200]: add test that skipScrubPath is empty array

**Review by doriaviram — CHANGES_REQUESTED** [review]

**Comment by efimk-lu** [comment]
@nirLumigo I really like the way you describe a PR in GitHub, good work!

File: src/parsers/eventParser.js [hunk 2]
```diff
@@ -255,3 +255,13 @@ export const parseEvent = event => {
...
   return event;
 };
+
+export const getSkipScrubPath = event => {
+  if (process.env.LUMIGO_SCRUB_KNOWN_SERVICES === 'true') {
+    return null;
+  }
+  if (isS3Event(event)) {
+    return ['Records', [], 's3', 'object', 'key'];
+  }
```
[the remark this answers is not in the document]
Comment (nirLumigo) [reply src/parsers/eventParser.js:265]: It's not part of the ticket, I don't know what is the issue there

**Review by doriaviram — APPROVED** [review]
